src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -461,6 +528,150 @@ public void swipeLeft(int times) {
         }
     }
 
+    /***
+     * Used to trigger iOS' gestural navigation back
+     * As of 8/29/2019, Android does not have this feature, but it may in the future
+     * More information on Android here: https://developer.android.com/preview/features/gesturalnav
+     */
+    public Locomotive swipeSystemBack() {
+        Dimension screen = getAppiumDriver().manage().window().getSize();
+        Point start = new Point(screen.getWidth() - 2, getYCenter());
+        Point end = new Point(2, getYCenter());
+
+        return swipe(start, end);
+    }
+    //endregion Directional swipes
+
+    //region longPress swipes
+    public Locomotive longPressSwipeCenter(SwipeElementDirection direction) {
+        return longPressSwipe(direction, /*element=*/null, SWIPE_DISTANCE, 0);
+    }
+
+    public Locomotive longPressSwipeCenter(SwipeElementDirection direction, int swipeDurationInMillis) {
+        return longPressSwipe(direction, /*element=*/null, SWIPE_DISTANCE, swipeDurationInMillis);
+    }
+
+    public Locomotive longPressSwipeCenter(SwipeElementDirection direction, short numberOfSwipes) {
```

Comment:
ditto about the overloads

Reply:
Good call. Removed the methods with a signature of only `short numberOfSwipes`.